api/v1alpha1/clienttrafficpolicy_types.go

```diff
@@ -104,6 +104,9 @@ type HeaderSettings struct {
 	// is encountered. The default action is to reject the request.
 	// +optional
 	WithUnderscoresAction *WithUnderscoresAction `json:"withUnderscoresAction,omitempty"`
+
+	// configure Envoy proxy to forward x-forwarded-client-cert (XFCC) HTTP header
```

Comment:
also need a way to decide which cert fields are being added https://www.envoyproxy.io/docs/envoy/latest/api-v3/extensions/filters/network/http_connection_manager/v3/http_connection_manager.proto#envoy-v3-api-msg-extensions-filters-network-http-connection-manager-v3-httpconnectionmanager-setcurrentclientcertdetails

Reply:
will add SetCurrentClientCertDetails

Reply:
thanks !
further thoughts on this 
1. this field should probably live inside `ClientValidation` since its tied to that feature
2. is there a better way to represent this (how to deal with the existing XFCC header and what to insert into it at this hop) rather than have a 1:1 API with Envoy ? any past work we can get inspiration from ?

Reply:
Istio expose 1:1 API with Envoy
https://istio.io/latest/docs/ops/configuration/traffic-management/network-topologies/#configuring-x-forwarded-client-cert-headers

but doesn't able to configure XFCC forwarded information

Reply:
resolving this since we will move it to Headers